### ISSUE TITLE
Add a functionality to lock every content entered by user

### ISSUE THREAD

**Issue by rapotek** [opencaching/opencaching-pl#2261]
This issue arises from OCPL "Terms of Use", but it may be applicable to another nodes as well. The OCPL "Terms of Use" states that:
> Zespół OC, po otrzymaniu urzędowego zawiadomienia lub uzyskania wiarygodnej wiadomości o bezprawnym charakterze danych umieszczonych w Serwisie przez Użytkownika lub związanej z nimi działalności, niezwłocznie uniemożliwi dostęp do tych danych (...)

The above citation means in short that OC TEAM should lock, ie. prevent from viewing every content entered by user which violates law or rules stated in "Terms of Use".
While locking a cache listing contents is possible by setting its state to "locked" and locking a log entry is possible by removing it and reverting this operation later, it should be possible to lock (to hide from a public view) a description in user profile and a text in stat picture as well, which can in theory contain forbidden contents.

**Comment by kojoty** [comment]
I don't think we need to provide such functionality - I don't know if anyone needs to put some "restricted' text as a user description or as a user stats text - this is theoretically possible but I don't remember such case. If it happens we can react by modification directly in DB.

**Comment by kojoty** [comment]
Rapotek, if you want to implement this it's up to you I respect your opinion and ideas but please consider it again:

- this is a corner case
- we have never had such issue
- there is no sense to modify the code and add options in UI for cases which never happen till now
- if it happens - let's say once a year - we can manually: 
    - save the original text
    - block the user account
    - change it in DB
    - investigate the issue
    - restore the original text if needed

In my opinion we should try to automate things which are useful for users, for octeam or for us - developers - I think this option is not useful for anyone as it will never be used. 

If we got such case once a year you spend much more time to implement this than we spend on DB modification in next 10 years... that's why I think it is OK in theory but this is waste of time and code complication

**Comment by rapotek** [comment]
> Rapotek, if you want to implement this it's up to you I respect your opinion and ideas but please consider it again:
> 
>     * this is a corner case
> 
>     * we have never had such issue
> 
>     * there is no sense to modify the code and add options in UI for cases which never happen till now
> 
>     * if it happens - let's say once a year - we can manually:
>       
>       * save the original text
>       * block the user account
>       * change it in DB
>       * investigate the issue
>       * restore the original text if needed
> 
> 
> In my opinion we should try to automate things which are useful for users, for octeam or for us - developers - I think this option is not useful for anyone as it will never be used.
> 
> If we got such case once a year you spend much more time to implement this than we spend on DB modification in next 10 years... that's why I think it is OK in theory but this is waste of time and code complication

Odpowiem po polsku, specjalnie: widząc to, co się dzieje u nas na forum, nie zdziwiłbym się, gdyby takie kwiatki zaczęły się pojawiać, bo ktoś komuś coś itd. Zgadzam się natomiast, że absolutnie nie jest to priorytet, chciałem tylko sprawę zasygnalizować.

I am closing this issue now, can be reopened when (if) necessary.

**Comment by piotr-juzwiak** [comment]
dzieki @rapotek 
Myślę, że mamy tak wiele otwartych fajnych pomysłów, że otwieranie takich tematów na zasadzie "a może się zdarzy" po prostu nie ma wiele sensu - jeśli jakieś trolle zaczną coś wstawiać w opisy i bedzie z tym duzo roboty to faktycznie to zautomatyzujemy ale mam nadzieję nikt na taki głupi pomysł nie wpadnie

**Comment by kojoty** [comment]
(z właściwego konta :))
dzieki @rapotek
Myślę, że mamy tak wiele otwartych fajnych pomysłów, że otwieranie takich tematów na zasadzie "a może się zdarzy" po prostu nie ma wiele sensu - jeśli jakieś trolle zaczną coś wstawiać w opisy i bedzie z tym duzo roboty to faktycznie to zautomatyzujemy ale mam nadzieję nikt na taki głupi pomysł nie wpadnie
